src/utils/formatDateToHumanReadable.ts

```diff
@@ -0,0 +1,18 @@
+export function formatDateToHumanReadable(
+  date: string,
+  timeZone: string,
+  locale = "en-GB",
```

Comment:
Set locale to `undefined` so it'll use the browsers / systems settings, dd/mm/yy :)